statsmodels/tsa/statespace/tbats.py

```diff
@@ -0,0 +1,1437 @@
+"""
+Trigonometric, Box-Cox, ARMA Error, Trend and Seasonal (TBATS) Model
+
+Author: Leoyzen Liu
+License: Simplified-BSD
+"""
+from __future__ import division, absolute_import, print_function
+
+from warnings import warn
+from statsmodels.compat.collections import OrderedDict
+
+import numpy as np
+from numpy.linalg import LinAlgError
+import statsmodels.api as sm
```

Comment:
AFAICS, this adds a circular import and causes the TravisCI test error.
inside the code we only import directly from the modules (or subpackages)

(I'm not sure what this circular import does and why it fails for some tests but not all.)